builtin/providers/aws/resource_aws_db_subnet_group.go

```diff
@@ -82,12 +83,17 @@ func resourceAwsDbSubnetGroupRead(d *schema.ResourceData, meta interface{}) erro
 	}
 
 	if len(describeResp.DBSubnetGroups) != 1 ||
```

Comment:
Looks like this conditional is doing nothing? Should we maybe return an error saying "More than one group returned"?

Also, for readability assigning the 0-indexed group to a var might be nice after this check.